ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_math.py

```diff
@@ -2137,3 +2137,36 @@ def test_paddle_stanh(
         scale_a=scale_a,
         scale_b=scale_b,
     )
+
+# amin
+@handle_frontend_test(
+    fn_tree="paddle.tensor.math.amin",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("valid"),
+        num_arrays=1,
+        allow_inf=True,
+        shared_dtype=True,
+    ),
```

Comment:
this is not the best strategy to be used as it doesn't generate the `axis` argument, and you should also test for `keepdim` argument